operations/node_management/Add_a_Standard_Rack_Node.md

```diff
@@ -15,29 +15,30 @@ These procedures are intended for trained technicians and support personnel only
 
 The example is this procedure adds a User Access Node \(UAN\) or compute node to an HPE Cray standard rack system. This example adds a node to rack number 3000 at U27.
 
-Procedures for updating the Hardware State Manager \(HSM\) or System Layout Service \(SLS\) are similar when adding additional compute nodes or User Application Nodes \(UANs\). The contents of the node object in the SLS are slightly different for each node type.
+Procedures for updating the Hardware State Manager \(HSM\) or System Layout Service \(SLS\) are similar when adding additional compute nodes or User Application Nodes \(UANs\). The contents of the node object in the SLS are slightly different
+for each node type.
 
 Refer to the OEM documentation for information about the node architecture, installation, and cabling.
 
 For this procedure, a new object must be created in the SLS and modifications will be required to the Slingshot HSN topology.
 
-### Prerequisites
+## Prerequisites
```

Comment:
If any scripts from the docs repo are used in the procedure, we should also add that as a prereq

Reply:
No scripts from the docs repo are used.